src/graph/LimitExecutor.cpp

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/LimitExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+LimitExecutor::LimitExecutor(Sentence *sentence, ExecutionContext *ectx) : TraverseExecutor(ectx) {
+    sentence_ = static_cast<LimitSentence*>(sentence);
+}
+
+
+Status LimitExecutor::prepare() {
+    skip_ = sentence_->skip();
+    if (skip_ < 0) {
+        return Status::Error("skip `%ld' is illegal", skip_);
+    }
+    count_ = sentence_->count();
+    if (count_ < 0) {
```

Comment:
i wonder if count_ == 0 has any meaning ? return all or return nothing.  You can add comment in the code to explain.